src/components/accounts/accounts-component/accounts-component.js

```diff
@@ -0,0 +1,105 @@
+import template from "./accounts-component.html";
+import accountItemTemplate from "./account-item.html";
+import { ButtonMoreComponent } from "../../button-more/button-more";
+import { AddAccountDialogComponent } from "../add-account-dialog/add-account-dialog";
+import { AccountListService } from "../../../services/account-service";
+
+export class AccountsComponent {
+  constructor(mountPoint, props) {
+    this.mountPoint = mountPoint;
+    this.props = props;
+  }
+
+  querySelectors() {
+    this.accountsRoot = this.mountPoint.querySelector(".accounts");
+    this.moreBtnMountPoints = this.mountPoint.querySelectorAll(
+      ".account__more-button"
+    );
+    this.addAccountButton = this.mountPoint.querySelector(
+      ".account__add-account-dialog-activation"
+    );
+    this.accountsList = this.mountPoint.querySelector(".account__list-items");
+  }
+
+  handleAddAccountConfirmed(account) {
+    AccountListService.add(account);
+    this.addAccountToHead(account);
+  }
+
+  initMoreBtns() {
+    this.moreBtnMountPoints = this.mountPoint.querySelectorAll(
+      ".account__more-button"
+    );
+    Array.from(this.moreBtnMountPoints).forEach(point => {
+      new ButtonMoreComponent(point, {
+        position: "right",
+        onDeleteClicked: this.handleDeleteClick.bind(this),
+        onEditClicked: this.handleEditClick.bind(this)
+      }).mount();
+    });
+  }
+
+  initAccounts(accounts) {
+    accounts.forEach(item => {
+      this.addAccount(item);
+    });
+  }
+
+  initData() {
+    this.accountsList.innerHTML = "";
+    AccountListService.get().then(accounts => {
+      if (!accounts) {
+        accounts = [];
+        AccountListService.set(accounts);
```

Comment:
please do not set data here -- it would be better if `AccountListService` would return empty list if there is no items there or just do like this

```js
AccountListService.get().then(accounts => this.initAccounts(accounts || []) })
```